activities/member-relations/information-about-founding-membership.md

```diff
@@ -0,0 +1,21 @@
+---
+type: Resource
+---
+
+# Information about Founding Membership template
+
+## Resource
+
+You can access the [template here](https://docs.google.com/document/d/165RXZmeU6klU_QmCR06ACvY8SAtN27de1nilqOmi8Nw/edit?usp=sharing).
+
+## Assumptions
+
+* This lightweight document that can be shared as minutes after an introductory meeting with interested public organizations
+* This resource is a link to a Google doc for now, but might change format
+* Any proposed changes to the content of this template must first be proposed on, and merged with, [the `master` branch of About](https://github.com/publiccodenet/about/tree/master) before being implemented here
+
+## Use and further development
+
+- [ ] Replace the “[highlighted text]” with customized text
+- [ ] Share with potential members
+- [ ] Revise template with feedback once shared with 2-3 potential members
```

Comment:
This should be tracked elsewhere and mixing usage and development instructions adds a lot of extra strain on both users and developers.